ascending navigation (#2227)
grommet/hpe-design-system: aries-site/src/pages/templates/navigation.mdx

## Types of Navigation

> Pages and content of a site or service are typically organized in an information hierarchy. Different types of navigation provide for traversing this hierarchy according to diverse user tasks. 
> There are 5 different types of navigational directions a user can maneuver in.

Reviewer: ```suggestion
Pages and content of a site or service are typically organized in an information hierarchy. Navigation provides various means for traversing this hierarchy depending on the user's task. 
There are 5 navigational directions in which a user can maneuver:
```

> **Drill Down**: used when the user wants to get more content for something or the user wants to see a higher level of content for the site/app than the current page shows.

Reviewer: ```suggestion
**Drill Down** - Used when the user wants to view greater detail on the current topic than the current page displays.
```

> **Side-to-Side**: used to go to the next or previous sibling of the current page, typically when the user is scanning across similar items.

Reviewer: ```suggestion
**Side-to-Side** - Used to visit to the next or previous sibling of the current page, typically when the user is scanning across similar items.
```

> **Reverse Navigation** (built in the web browser): used to go back to the previous visited page utilizing the browsers built-in back button

Reviewer: ```suggestion
**Reverse Navigation** - Used to return to the previous visited page using the browser's built-in back button.
```

> **Matrix**: used to go to a different place in the content hierarchy that isn't a direct parent or child of the current page in the content hierarchy.

Reviewer: ```suggestion
**Matrix** - Used to "jump" to a different place in the information hierarchy which isn't a direct parent or child of the current page.
```

> **Ascending Navigation**: used to direct the user up the hierarchy to the parent page when on a child page.

Reviewer: ```suggestion
**Ascending Navigation** - Used to allow the user to move up the information hierarchy from a child page to its parent.
```

**Follow the points below for successful navigation:**
- Identify the problem.
- Understand what the user’s role is and common processes followed.

> - Who is this being intended for and determine thee most common tasks.

Reviewer: What is "this"?

```suggestion
- Identify for whom this is intended and determine their most common tasks.
```

> To learn more on what makes a successful Navigation, [check out the Figma file.](https://www.figma.com/file/b6hiokanRCwNnc93PpHdgQ/HPE-Design-System-Navigation?node-id=0%3A1)

Reviewer: ```suggestion
For more tips on how to implement successful navigation, [check out this Figma file](https://www.figma.com/file/b6hiokanRCwNnc93PpHdgQ/HPE-Design-System-Navigation?node-id=0%3A1).
```
